explorer.schema.yaml

```
@@ -0,0 +1,973 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+title: Diffix Explorer Output Schema
```

Maybe we should name this API V1 or something to that effect? 

```suggestion
title: Diffix Explorer API v1 Output Schema
```